.tekton/.currency/currency-pipeline.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: ruby-tracer-currency-pipeline
+spec:
+  params:
+  - name: revision
+    type: string
+  - name: ruby_3_3_image_digest
+    type: string
+    description: Ruby 3.3 image digest
+    default: "sha256:5cf0004738f54bd67e4c4316394208ca38a6726eda7a1b0586d95601aad86e5d"
```

Comment:
Its still from docker. There are a lot of changes that needs to be done for this. Will raise a separate PR just like one for python for registry change will that work?

Reply:
I think it will be good to make this change now. Then you have one less line to change in the future. 😄  

Please also add a comment before the image line with the full URL of the image so we know which one is being used.

Reply:
Sure sure will do that!